src/it/saxon/src/test/java/com/jcabi/saxon/SaxonSampleTest.java

```diff
@@ -56,13 +57,16 @@
      */
     @Test
     public void validatesInMultipleThreads() throws Exception {
+        final int zerotime = 0;
+        final int tentimes = 10;
+        final int hundredtimes = 100;
+        final int fiftytimes = 50;
```

Comment:
@tmjee and this `tcount` ... not a perfect one ... find a better name if you want ... now all these variables have some meaning, instead of just translating the value in words